src/tests/cucumber/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+file(WRITE "${CMAKE_CURRENT_SOURCE_DIR}/latest_binary_dir.txt" "${CMAKE_BINARY_DIR}")
```

Comment:
```suggestion
file(GENERATE OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/conf_$<CONFIG>.yaml CONTENT "antares-solver : $<TARGET_FILE:antares-solver>")
```

Reply:
this will produce a yaml file with a key-value pair :
antares-solver : /path/to/antares-solver